src/v/redpanda/admin_server.cc

```diff
@@ -666,3 +675,86 @@ void admin_server::register_partition_routes() {
           co_return ss::json::json_void();
       });
 }
+
+void admin_server::register_hbadger_routes() {
+    /*
```

Comment:
we need to further protect this via `#ifndef NDEBUG return; #endif` - we can't release this to a prod cluster too risky yet. let's get confidence in honey badger. even if the `is_enabled` is there. this is our last line of defense.